File: src/main/clients/openrgb/effects/Static.ts
import { AbstractEffect } from '@main/clients/openrgb/effects/base/AbstractEffect';

class Static extends AbstractEffect {
  public constructor() {
    super('Static', true);
  }

  protected async applyEffect(): Promise<void> {
    this.devices.forEach((dev) => {
      this.setColors(dev, Array(dev.colors.length).fill(this.color));
    });
    this.hasFinished = true;
  }
}

export const staticEffect = new Static();

File: src/clients/openrgb/effects/static.py
from .base.abstract_effect import AbstractEffect
from ....utils.singleton import singleton


@singleton
class StaticEffect(AbstractEffect):
    def __init__(self):
        super().__init__("Static", "#FF0000")

    def apply_effect(self):
        for dev in self.devices:
            self.set_colors(dev, [self._color] * len(dev.leds))


static_effect = StaticEffect()
